index.bs

```diff
@@ -2890,6 +2890,15 @@ To <dfn>delete sources for unexpired destination limit</dfn> given a [=set=] of
         and |report|'s [=event-level report/trigger time=] is greater than or equal to |now|:
         1. [=set/Append=] |report|'s [=event-level report/report ID=] to |deletedEventLevelReports|.
         1. [=set/Remove=] |report| from the [=event-level report cache=].
+
+    Note: The privacy concern on leaking browsing history of destinations
```

Comment:
IMO this sentence is a bit hard to parse, especially because it's unclear what "the privacy concern" actually is.

Reply:
The privacy concern is `leaking browsing history of destinations...`. Please let me know if you have better suggestion to rephrase. Thanks.

Reply:
Updated, thanks.